src/test-utils/dom/app-layout/index.ts

```diff
@@ -39,6 +47,14 @@ export default class AppLayoutWrapper extends ComponentWrapper {
     return this.findByClassName(testutilStyles.tools)!;
   }
 
+  findOpenToolsPanel(): ElementWrapper | null {
```

Comment:
**This is non blocking:** Should we generally consider making conditions into parameters?
```tsx
findToolsPanel({ isOpen: true });
```

Reply:
`findTools` does not have `ElementWrapper  | null` in the return value type. `findToolsPanel({ isOpen: true })` may return null. 

If I am going to build this test util API from scratch, I could go your suggestion, but not in the existing code

Reply:
I don't think we should fix this here either 👍